internal/envoygateway/config/config.go

```diff
@@ -37,19 +38,22 @@ type Server struct {
 	// Logger is the logr implementation used by Envoy Gateway.
 	Logger logging.Logger
 	// Elected chan is used to signal what a leader is elected
-	Elected chan struct{}
+	Elected *sync.WaitGroup
```

Comment:
why we need a pointer here?

Reply:
internal/infrastructure/runner/runner.go:35:25: copylocks: literal copies lock value from *cfg: github.com/envoyproxy/gateway/internal/infrastructure/runner.Config contains github.com/envoyproxy/gateway/internal/envoygateway/config.Server contains sync.WaitGroup contains sync.noCopy (govet)
        return &Runner{Config: *cfg}